### PR TITLE
Add tokens component docs

### PR DESCRIPTION
**Pull request by blackbaud-johnly**
Addresses https://github.com/blackbaud/skyux2/issues/1849 and https://github.com/blackbaud/skyux2/issues/1848.

### CODE REVIEW

File: src/app/components/tokens/index.html [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+      propertyName="focusable"
+      defaultValue="true"
+      isOptional="true"
+    >
+      Indicates whether users can focus on items in the list using the Tab key. This does not affect the ability of users to select tokens, dismiss tokens, or navigate through the list with the arrow keys.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="messageStream"
+    >
+      Specifies ...
```
Comment (blackbaud-conorwright) [inline src/app/components/tokens/index.html]: This is an observable of SkyTokenMessageTypes that can send certain commands to the tokens component. The options are multiple focusing options and one to remove the active token.
Options are: FocusLastToken, FocusActiveToken, FocusPreviousToken, FocusNextToken, and RemoveActiveToken.

File: src/app/components/tokens/index.html [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+  <sky-demo-page-properties sectionHeading="Tokens list events">
+    <sky-demo-page-property
+      propertyName="tokensChange"
+    >
+      Fires when the tokens in the list change. This event emits an array of the tokens in the updated list.
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="focusIndexOverRange"
+    >
+      Fires when ...
```
Comment (blackbaud-conorwright) [inline src/app/components/tokens/index.html]: Fires when the active token navigation tries to move past the final token in the list

File: src/app/components/tokens/index.html [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+    </sky-demo-page-property>
+    <sky-demo-page-property
+      propertyName="focusIndexUnderRange"
```
Comment (blackbaud-conorwright) [inline src/app/components/tokens/index.html]: Fires when the active token navigation tries to move before the first token in the list

**Comment by blackbaud-johnly** [comment]
@blackbaud-conorwright: I updated the PR to incorporate all of your feedback. Thanks again! ... @Blackbaud-SteveBrush: I removed the "HOLD" from the PR title and refreshed it with the latest changes from `master`. Can you please review the updated PR since Conor took the first pass through it? Thanks in advance.

File: src/app/components/tokens/index.html [hunk 4]
```diff
@@ -0,0 +1,156 @@
...
+    >
+      Specifies an array of tokens to include in the list. 
+    </sky-demo-page-property>
+  </sky-demo-page-properties>
+
+  <sky-demo-page-properties sectionHeading="Tokens list events">
+    <sky-demo-page-property
+      propertyName="focusIndexOverRange"
+    >
+      Fires when users navigate through the token list with the Tab key and attempt to move past the final token in the list.
```
Comment (Blackbaud-SteveBrush) [inline src/app/components/tokens/index.html]: Sometimes it reads "tokens list" and sometimes it reads "token list". Should they be consistent?

File: src/app/components/tokens/index.html [hunk 5]
```diff
@@ -0,0 +1,156 @@
+<sky-demo-page
+  pageTitle="Tokens"
+  skyuxModule="SkyTokensModule"
+>
+
+  <sky-demo-page-summary>
+    <p>
+      The tokens component displays a series of specified objects for users to interact with. Users can navigate through the list of tokens and remove tokens from the list. The tokens component's includes a message stream to tokens to the list.
```
[the remark this answers is not in the document]
Comment (blackbaud-johnly) [reply src/app/components/tokens/index.html]: This should have been "The tokens component includes a message stream to place focus on tokens or remove tokens from the list."

File: src/app/components/tokens/index.html (hunk 4 above)
Comment (blackbaud-johnly) [reply src/app/components/tokens/index.html]: It should be consistent. Thanks for catching that. I'll update all instances of "token list" to be "tokens list."

**Comment by blackbaud-johnly** [comment]
@Blackbaud-SteveBrush, I updated the PR to address your feedback. Can you please approve when you get a chance? Thanks!

**Comment by blackbaud-johnly** [comment]
@Blackbaud-SteveBrush. does the updated PR look good to you?

**Pull request merged**